src/widgets/widget_definitions.rs

```diff
@@ -145,6 +146,17 @@ pub(super) fn resolve_orientable_attrs(bargs: &mut BuilderArgs, gtk_widget: &gtk
 
 // concrete widgets
 
+/// @widget color button
+fn build_gtk_color_button(bargs: &mut BuilderArgs) -> Result<gtk::ColorButton> {
+    let gtk_widget = gtk::ColorButtonBuilder::new().build();
+    resolve_block!(bargs, gtk_widget, {
+        // @prop use-alpha - bool to wether or not use alpha
+        prop(alpha: as_bool) {gtk_widget.set_use_alpha(alpha);}
```

Comment:
```suggestion
        prop(use_alpha: as_bool) {gtk_widget.set_use_alpha(alpha);}
```

stay true to what your documentation says ;D